### DISCUSSION TITLE
Code autocompletion

### DISCUSSION THREAD

**Discussion by szafran98** [gleam-lang/gleam#2000] [Questions & support] [answered]
Hello, i wanted to try this language so i've installed it and VSC plugin, but code completion doesn't work. I'm not using VSC daily so i've checked also neovim with lspconfig and there is no any difference. It makes learning this interesting language much harder. Does the plugin support code autocompletion?

**Comment by lpil** [comment, marked as answer]
Hello! We currently have autocompletion of imports but nothing else. This is a feature we are actively working on but requires some architectural changes to the compiler to make it suitable for this form of interactive use. We hope to have initial autocompletion soon.
